CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Pull Requests
+
+ - Include a diff of the code of the current Minecraft version with and without the changes. An easy way to create this diff:
+   - Use [MCPConfig](https://github.com/MinecraftForge/MCPConfig) to decompile MC
```

Comment:
This would be fine if MCPConfig contained usage instructions, but as it is, it's like saying "here's this thing that will do a thing, work it out"

Just needs a quick sublist of something like 
- clone
- run gradle task X
- output is at Y